io/net/bridge.py

```diff
@@ -96,6 +97,10 @@ def test_bridge_delete(self):
         '''
         self.check_failure('ip link del dev %s' % self.bridge_interface)
         # TODO:need to get this functionality into avocado utils interfcae.py
-        path = "/etc/sysconfig/network-scripts/ifcfg-%s" \
-               % self.bridge_interface
+        detected_distro = distro.detect()
+        if detected_distro.name == 'rhel':
+            path = "/etc/sysconfig/network-scripts/ifcfg-%s" \
+                   % self.bridge_interface
+        elif detected_distro.name == "SuSE":
+            path = "/etc/sysconfig/network/ifcfg-%s" % self.bridge_interface
```

Comment:
@smruti77 Can this be handled in utils ?
See the TODO in the line above.

Reply:
@narasimhan-v yes,we decided to add later so i thought to send here first.as in suse if we run bridge error is coming.